robot/Cumulus/resources/ObjectMangerPageObject.py

```diff
@@ -187,4 +189,56 @@ def add_picklist_values(self, **kwargs):
 				self.selenium.get_webelement(picklist_txtarea).send_keys(Keys.ENTER)
 			self.selenium.click_element(picklist_save)
 		else:
-			raise Exception(f"Field not found")
\ No newline at end of file
+			raise Exception(f"Field not found")
+
+	def rename_object_field(self,object_name,field_name,new_name):
+		"""Renames the field with new name if a field doesn't exist with that name"""
+		save_button=npsp_lex_locators['object_manager']['button'].format("Save")
+		self.open_fields_and_relationships(object_name)
+		search_button = npsp_lex_locators['object_manager']['input'].format("globalQuickfind")
+		self.selenium.wait_until_page_contains_element(search_button,60)
+		self.selenium.get_webelement(search_button).send_keys(new_name)
+		self.selenium.get_webelement(search_button).send_keys(Keys.ENTER)
+		time.sleep(1)
+		self.salesforce.wait_until_loading_is_complete()
+		field_exists = self.selenium.get_webelements(npsp_lex_locators['object_manager']['search_result'].format(new_name))
+		if not field_exists:
+			self.selenium.clear_element_text(search_button)
+			self.selenium.get_webelement(search_button).send_keys(field_name)
+			self.selenium.get_webelement(search_button).send_keys(Keys.ENTER)
+			time.sleep(1)
+			self.salesforce.wait_until_loading_is_complete()
+			search_result = npsp_lex_locators['object_manager']['search_result'].format(field_name)
+			self.selenium.click_element(search_result)
+			self.npsp.select_frame_and_click_element("Account Custom Field","object_manager.button","Edit")
+			self.selenium.unselect_frame()
+			self.npsp.wait_for_locator('frame_new', 'Edit Account Custom Field', 'Edit Account Custom Field')
+			self.npsp.choose_frame('Edit Account Custom Field')
+			self.selenium.wait_until_page_contains_element(save_button)
+			locator = self.salesforce._get_input_field_locator('Field Label')
+			element = self.selenium.get_webelement(locator)
+			self.selenium.driver.execute_script("arguments[0].setAttribute(arguments[1], arguments[2]);", element, "onChange", "")
```

Comment:
What does this jS execution block do ?

Reply:
it is going to disable the alert that popsup when u change the value in field